apps/emqx/src/emqx_persistent_session_ds/emqx_persistent_session_ds_state.erl

```diff
@@ -534,6 +544,12 @@ n_awaiting_rel(Rec) ->
 
 %%
 
+-spec iterate(iter(K, V)) -> {K, V, iter(K, V)} | none.
+iterate(It0) ->
```

Comment:
Tiny nit: maybe:

```suggestion
next(It0) ->
```

?

Reply:
Yes, I guess this name is better for the sake of naming consistency. We use `next` everywhere else.